assets/css/main.scss

```diff
@@ -874,228 +878,122 @@ p .btn--aio {
   }
 }
 
-.license-comparison {
-  display: flex;
-  align-items: center;
-
-  .comparison-img {
-    flex: 30%;
-    padding: 0.5em;
-  }
-  .comparison-text {
-    flex: 30%;
-    padding-right: 0.5em;
-    padding-left: 0.5em;
-    text-align: center;
-  }
-}
-
-
-
-
-.toggle, .toggler {
-  display: inline-block;
-  vertical-align: middle;
-  margin: 10px;
-}
-
-.toggler {
-  color: #ddd;
-  transition: .2s;
-  font-weight: bold;
-}
-
-.toggler--is-active {
-  color: #000;
-}
-
-.b {
-  display: block;
-}
+// .license-comparison {
+//   display: flex;
+//   align-items: center;
 
-.toggle {
-  position: relative;
-  width: 80px;
-  height: 35px;
-  border-radius: 100px;
-  background-color: #000;
-  overflow: hidden;
-  box-shadow: inset 0 0 2px 1px rgba(0, 0, 0, 0.05);
-}
-
-.check {
-  position: absolute;
-  display: block;
-  cursor: pointer;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  opacity: 0;
-  z-index: 6;
-}
-
-.check:checked ~ .switch {
-  right: 2px;
-  left: 57.5%;
-  transition: 0.25s cubic-bezier(0.785, 0.135, 0.15, 0.86);
-  transition-property: left, right;
-  transition-delay: .08s, 0s;
-}
-
-.switch {
-  position: absolute;
-  left: 2px;
-  top: 2px;
-  bottom: 2px;
-  right: 57.5%;
-  background-color: #fff;
-  border-radius: 36px;
-  z-index: 1;
-  transition: 0.25s cubic-bezier(0.785, 0.135, 0.15, 0.86);
-  transition-property: left, right;
-  transition-delay: 0s, .08s;
-  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
-}
+//   .comparison-img {
```

Comment:
Feel free to delete commented-out code to keep source files clean when making a PR. We have the git commit history in case we want to recover them.